spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/AbstractMessageListenerContainer.java

```diff
@@ -1068,7 +1090,10 @@ public void shutdown() {
 	 *
 	 * @throws Exception Any Exception.
 	 */
-	protected abstract void doInitialize() throws Exception;
+	protected void doInitialize() throws Exception {
+		checkMissingQueuesFatalFromProperty();
+		checkPossibleAuthenticationFailureFatalFromProperty();
+	}
 
```

Comment:
Why not do this in `initialize` rather than removing the abstract method (which could be dangerous)?